app/views/publisher_information/publisher_updates.govspeak.md

```diff
@@ -1,3 +1,6 @@
+### Backdating <span class="govuk-caption-m">28 June 2019</span>
+You can now backdate the First Published date on content if it has previously been available on another website. It is important that old information is not presented a new.
```

Comment:
"is not presented a" = > "is not presented as"

Reply:
Should we mention it's only available on first editions?

Reply:
fixed typo, updated How to page to explain first editions.